src/js/src/components/SearchFacetOptions.vue

```diff
@@ -77,17 +77,30 @@ export default {
       let appliedFacets = this.searchAppliedFacets.join('')
       this.addSpecificRequestedFacets({facet:facetArea, query:structuredQuery, appliedFacets:appliedFacets})
     },
-    applyFacet(facetCategory, facet) {
+    applyFacet(facetCategory, facet, event) {
       let facetAllreadyApplied = false
       let newFacet = '&fq=' + facetCategory + ':"' + facet + '"'
       this.searchAppliedFacets.forEach((facet) => {
         facet === newFacet ? facetAllreadyApplied = true : null
       })
       if(!facetAllreadyApplied) {
-        this.updateSolrSettingOffset(0)
-        this.addToSearchAppliedFacets(newFacet)
-        this.$_pushSearchHistory('Search', this.query, this.searchAppliedFacets, this.solrSettings)
-    }
+          if(event.ctrlKey === true) {
+            const localSolrSettings = JSON.parse(JSON.stringify(this.solrSettings))
+            const localFacets = [...this.searchAppliedFacets]
+            localFacets.push(newFacet)
+            localSolrSettings.offset = 0
+            console.log(this.$_getResolvedUrl('Search', this.query, localFacets, localSolrSettings))
```

Comment:
We don't want this console.log in production

Reply:
Missed one, sorry.